Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -43,40 +42,30 @@ However, this may require extensions you don't need or omit extensions you do
 need, so be sure to update the required extensions as described in the next
 sections.
 
-The :file:`composer.json` in the Base distribution includes a scripts section:
-
-.. code-block:: json
-   :caption: /composer.json
-
-   {
-      "scripts": {
-         "typo3-cms-scripts": [
-            "typo3cms install:generatepackagestates",
-            "typo3cms install:fixfolderstructure"
-         ],
-         "post-autoload-dump": [
-            "@typo3-cms-scripts"
-         ]
-      }
-   }
+You can also create the :file:`composer.json` file via a :bash:`composer init` command.
```

Comment:
Please link the "composer.json" in line 34 with https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/ExtensionArchitecture/FileStructure/ComposerJson.html for more information.